File: app/src/main/java/com/esorokin/lantector/model/network/data/detect/DetectLanguageResponse.java
package com.esorokin.lantector.model.network.data.detect;

import com.esorokin.lantector.model.network.data.BaseResponse;

import org.simpleframework.xml.Element;
import org.simpleframework.xml.Root;

import lombok.Getter;

@Root(name = "results", strict = false)
@Getter
public class DetectLanguageResponse extends BaseResponse {
	@Element(name = "language", required = false)
	private String language;
}

File: app/src/main/java/com/esorokin/lantector/model/network/data/detectlanguage/DetectLanguageResponse.kt
package com.esorokin.lantector.model.network.data.detectlanguage

import com.esorokin.lantector.model.network.data.BaseResponse
import org.simpleframework.xml.Element
import org.simpleframework.xml.Root

@Root(name = "results", strict = false)
class DetectLanguageResponse(
        @field:Element(name = "language", required = false)
        var language: String = "russian") : BaseResponse()